### PR TITLE
co2 layout setup

### PR DESCRIPTION
**Pull request by zmyslek**

### CODE REVIEW

File: src/routes/co2/+page.svelte
Comment (JoeriH06) [inline src/routes/co2/+page.svelte:1]: Make the script --> eithr without the comments, or explain what the comments mean
Then we use tailwind, there should not be any <style> make it mobile first as always with the classes that are given in the tailwind docs

File: src/routes/+layout.svelte
Comment (JoeriH06) [inline src/routes/+layout.svelte:1]: Why from auto to 0???

File: package.json
Comment (JoeriH06) [inline package.json:1]: why is this extra dependencie added? i dont think we need it, because we use tailwind as our "canvas"

**Review by JoeriH06 — CHANGES_REQUESTED** [review]
look at the comments

File: src/routes/+layout.svelte
Comment (zmyslek) [reply src/routes/+layout.svelte:1]: cause 0 is needed and auto does not ensure that

File: src/routes/co2/+page.svelte
Comment (zmyslek) [reply src/routes/co2/+page.svelte:1]: done

**Pull request closed without merging**